packages/next-tinacms-markdown/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "next-tinacms-markdown",
+  "version": "0.1.0-alpha.0",
+  "license": "Apache-2.0",
+  "author": "Nolan Phillips",
+  "main": "dist/index.js",
+  "module": "dist/next-tinacms-markdown.esm.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "watch": "tsdx watch",
+    "build": "tsdx build",
+    "dev": "tsdx build",
+    "test": "tsdx test --env=jsdom",
+    "lint": "tsdx lint"
+  },
+  "peerDependencies": {
+    "next": ">=9",
+    "react": ">=16",
+    "tinacms": ">=0.11"
+  },
+  "dependencies": {
+    "next": ">=9",
```

Comment:
`next`, `react`, and `tinacms` should just be in `peerDependencies` and `devDependencies`.

I assume this came from `next-tinacms-json`?

Reply:
oh yeah I just copied it over

Reply:
is `gray-matter` okay being in `dependencies`? I saw that's how it was configured for `gatsby-tinacms-remark`

Reply:
Yup!

Here's a pseudocode rule of thumb:
```
if package_X is a plugin for package_Y
then package_Y is a peerDependency of package_X
```

Reply:
The build fails now without `tinacms` in `dependencies`...which makes sense. Thoughts on how to fix?

Reply:
If it's in `peerDependencies` ti should also be in `devDependencies`